test/test_distributions.py

```diff
@@ -99,7 +99,7 @@ def sample(self, key, sample_shape=()):
 }
 
 CONTINUOUS = [
-    T(dist.Beta, 1., 2.),
+    T(dist.Beta, 0.2, 1.1),
```

Comment:
nothing special, I just changed this for pathwise testing but forgot to reset the values =.=